tests/testcases/Group10-Vulnerability/10-03-Clair-scan-0-package-image.md

```diff
@@ -0,0 +1,24 @@
+Test 10-03 Clair scan 0 package image
+=======
+  
+# Purpose:  
+To verify clair scan result of a 0 package image.  
+  
+# References:  
+User guide  
+  
+# Environment:  
+* This test requires that a Harbor instance is running and available.  
+* Harbor is installed with clair enable.  
+* A linux host with Docker CLI installed.  
```

Comment:
Again, an important prereq is:
A prepreq is Clair's Vuln DB has updated successfully.